### ISSUE TITLE
Support in Webpack

### ISSUE THREAD

**Issue by nathanchase** [miguelzakharia/aurelia-google-analytics#8]
I tried using this plugin alongside the [Aurelia Webpack-based skeleton](https://github.com/aurelia/skeleton-navigation/tree/master/skeleton-es2016-webpack), and received the following errors:

`Uncaught (in promise) Error: Cannot find module './aurelia-google-analytics'.
    at eval (webpack:///./src_^\.\/.*$?:263:41)
    at webpackContextResolve (webpack:///./src_^\.\/.*$?:263:89)
    at webpackContext (webpack:///./src_^\.\/.*$?:260:29)
    at eval (webpack:///./~/aurelia-loader-webpack/dist/commonjs/aurelia-loader-webpack.js?:104:79)
    at http://localhost:9001/bundle.js:136:24
    at Function.requireEnsure [as e] (http://localhost:9001/bundle.js:597:29)
    at Function.ensure [as e] (http://localhost:9001/bundle.js:134:33)
    at eval (webpack:///./~/aurelia-loader-webpack/dist/commonjs/aurelia-loader-webpack.js?:103:31)
    at WebpackLoader._import (webpack:///./~/aurelia-loader-webpack/dist/commonjs/aurelia-loader-webpack.js?:98:12)
    at eval (webpack:///./~/aurelia-loader-webpack/dist/commonjs/aurelia-loader-webpack.js?:154:16)`

`./src ^\.\/.*$
Module not found: Error: Cannot resolve module 'aurelia-google-analytics' in D:\webpack\src`

`./~/aurelia-google-analytics/dist/es6/analytics.js
Module parse failed: D:\webpack\node_modules\aurelia-google-analytics\dist\es6\analytics.js Unexpected character '@' (89:0)
You may need an appropriate loader to handle this file type.
SyntaxError: Unexpected character '@' (89:0)
    at Parser.pp.raise (D:\webpack\node_modules\acorn\dist\acorn.js:920:13)
    at Parser.pp.getTokenFromCode (D:\webpack\node_modules\acorn\dist\acorn.js:2813:8)
    at Parser.pp.readToken (D:\webpack\node_modules\acorn\dist\acorn.js:2508:15)
    at Parser.pp.nextToken (D:\webpack\node_modules\acorn\dist\acorn.js:2500:71)
    at Parser.pp.next (D:\webpack\node_modules\acorn\dist\acorn.js:2449:8)
    at Parser.pp.eat (D:\webpack\node_modules\acorn\dist\acorn.js:1420:10)
    at Parser.pp.semicolon (D:\webpack\node_modules\acorn\dist\acorn.js:1462:13)
    at Parser.pp.parseVarStatement (D:\webpack\node_modules\acorn\dist\acorn.js:1930:8)
    at Parser.pp.parseStatement (D:\webpack\node_modules\acorn\dist\acorn.js:1729:19)
    at Parser.pp.parseTopLevel (D:\webpack\node_modules\acorn\dist\acorn.js:1648:21)
 @ ./src ^\.\/.*$`

I have the plugin added to my `webpack.config.js` as follows:

`plugins: [
     new AureliaWebpackPlugin({
         includeSubModules: [
           { moduleId: 'aurelia-google-analytics' }
         ]
     })
]`

and in my main.js:

`aurelia.use
        .standardConfiguration()
        .developmentLogging()
        .plugin('aurelia-google-analytics', config => {
            config.init('<Your Tracker ID>');
            config.attach({
                logging: {
                    enabled: true // Set to`true`to have some log messages appear in the browser console.
                },
                pageTracking: {
                    enabled: true // Set to`false`to disable in non-production environments.
                },
                clickTracking: {
                    enabled: true, // Set to`false`to disable in non-production environments.
                    filter: (element) => {
                        return element instanceof HTMLElement &&
                        (element.nodeName.toLowerCase() === 'a' ||
                            element.nodeName.toLowerCase() === 'button');
                    }
                }
            });
        })
});`

Any help getting this plugin working would be much appreciated! Thanks!

**Comment by miguelzakharia** [comment]
Ok, I have a temporary solution that will get you going until I (or someone) can come up with something better.
1. `cp node_modules/aurelia-google-analytics/dist/commonjs/* node_modules/aurelia-google-analytics/dist`
2. In `webpack.config.js` replace `plugins` with this code:

```
plugins: [
    new AureliaWebpackPlugin({
        includeSubModules: [
          { moduleId: 'aurelia-google-analytics' }
        ],
        contextMap: {
          'aurelia-google-analytics': 'node_modules/aurelia-google-analytics/dist/commonjs/index.js'
        }
      }),
    new ProvidePlugin({
      Promise: 'bluebird',
      jQuery: 'jquery'
    })
  ]
```

Now `npm run build` should work. I'm going to leave this issue open until there's a less hacky solution.

**Comment by nathanchase** [comment]
Awesome, that works. Thanks for the quick fix! 

FYI, it works without the 

```
new ProvidePlugin({
      Promise: 'bluebird',
      jQuery: 'jquery'
    })
```

block.

**Comment by nathanchase** [comment]
Any possibility of a more concrete fix with a less hacky solution?

**Comment by miguelzakharia** [comment]
I've been really busy with work, so I didn't have a chance to research a solution. I'm going to spend some time on it today.

**Comment by miguelzakharia** [comment]
I just published v1.0.4 which fixes this issue. Please let me know if you have any problems and I'll reopen the issue.

**Comment by nathanchase** [comment]
Works well here - thanks! Much appreciated.
